src/components/Navbar.tsx

```diff
@@ -14,7 +14,7 @@ export default function Navbar() {
   return (
     <>
       <div className='Navbar' style={{backgroundColor: theme === 'light' ? 'var(--bg-color)' : 'var(--bg-color)'}}>
-        <img className='navbar-logo' src={theme === 'light' ? DCCLogoLight : DCCLogoDark} alt="DCC Logo" />
+        <img className='navbar-logo' src={theme === 'light' ? DCCLogoLight : DCCLogoDark} alt="DCC Logo" onClick={() => window.location.href = '/'} style={{ cursor: 'pointer' }} />
```

Comment:
Is it possible to use the React router to handle this location change?

Reply:
React router navigation implemented, it's much better functionality. Great suggestion!